packages/frontend-main/src/composables/useExternalLink.ts

```diff
@@ -0,0 +1,18 @@
+import { externalLinks } from '@/config/externalLinks';
+
+type ExternalLinkKey = keyof typeof externalLinks;
+
+export function useExternalLink() {
+  function openExternalLink(key: ExternalLinkKey) {
```

Comment:
For this use case, it does not seem necessary. We should just use:

`<a :href="externalLinks.siteUrl" target="_blank" ...`

It's even better for accessibility.